app/models/ems_event.rb

```diff
@@ -246,8 +245,6 @@ def self.create_event(event)
       :timestamp   => event[:timestamp],
       :chain_id    => event[:chain_id],
       :ems_id      => event[:ems_id],
-      :target_id   => event[:target_id],
-      :target_type => event[:target_type],
```

Comment:
The commit and PR doesn't really describe "why" we can remove this.  Can you explain why?

Reply:
yes, I added these a couple of week ago to distinguish between MAS(MiqAlertStatuses) generated from the same miq alert and targeted at different objects. I then found out it is not enough - we can event have several different alerts targeted at the same object!
(e.g alerts for different containers that are linked to a node)

That's why I have to use the ems_ref. Since i'm using the ems_ref the target became redundant. As this query happens a lot and on a large data set I think it is better simplified.

Reply:
@moolitayer Can you put this information in the commit?  We can see what was changed when looking at git log but the "why" was missing.  Always ask yourself "why am I making this change" when writing the commit message.  The information you have above is perfect to add to the commit message.  Thanks!